base_local_planner/src/latched_stop_rotate_controller.cpp

```diff
@@ -135,6 +136,16 @@ bool LatchedStopRotateController::stopWithAccLimits(const tf::Stamped<tf::Pose>&
     boost::function<bool (Eigen::Vector3f pos,
                           Eigen::Vector3f vel,
                           Eigen::Vector3f vel_samples)> obstacle_check) {
+  if (stopping_scaling_percent_ <= 0.0) {
+    ROS_INFO_NAMED("latched_stop_rotate", "Slowing down...immediately");
+    ROS_DEBUG_NAMED("latched_stop_rotate", "Slowing down...immediately");
```

Comment:
If debug is enabled, info won't be needed. Should only have 1 of these.

Reply:
The info blocks were just added for ease for testing with Pete, his only availability was during a block of meetings for me, so I needed to go through the logs instead of live debuging